CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_apple_search_ads_source v0.2.1
```

Comment:
This looks great, would you also be able to add a [Contribors](https://github.com/fivetran/dbt_ad_reporting/blob/main/CHANGELOG.md#contributors) section to this release to give yourself contributor credit for your PR. You can see how this has been done within a few of our other package CHANGELOGS.

Reply:
Just added the section!:)